src/WSServerCon.cpp

```diff
@@ -684,6 +685,63 @@ void WSServerCon::processMessage(const QString &message)
             sendJsonMessage(oroot);
         });
     }
+    else
+    {
+        //Only ble related commands
+        MPDeviceBleImpl *bleImpl = mpdevice->ble();
```

Comment:
check the validity of `bleImpl`. If it's nullptr because the device is not a ble, it will crash